regions/shapes/tests/test_circle.py

```diff
@@ -46,3 +46,18 @@ def test_plot():
     assert p.get_facecolor() == (1, 0, 0, 0.6)
 
     c.plot(ax, alpha=0.6)
+
+def test_transformation():
+    
+    skycoord = SkyCoord(3 * u.deg, 4 * u.deg, frame='galactic')
+    skycircle = CircleSkyRegion(skycoord, 2 * u.arcsec)
+    
+    headerfile = get_pkg_data_filename('data/example_header.fits')
+    h = getheader(headerfile)
+    wcs = WCS(h)
+    pixcircle = skycircle.to_pixel(wcs)
+    assert (pixcircle.center == (-50.5, 299.5)).all()
```

Comment:
Test is passing, but still: I never write exactly float compares, always `numpy.testing.asser_allclose`. Suggest you use that here.